### DISCUSSION TITLE
Skype

### DISCUSSION THREAD

**Discussion by caveman927** [community/community#109202] [Code Security]
### Select Topic Area

Bug

### Body

I need to remove my phone number from Skype and telegram 
And deactivate both

**Comment by kyuna0312** [comment]
Sure, I can guide you through the process.

For Skype:

    Open Skype on your device.
    Click on your profile picture or your name in the upper-left corner to open your profile.
    In the profile section, locate your phone number.
    Click on your phone number, and you should see an option to remove it.
    Follow the prompts to remove your phone number from Skype.

To deactivate your Skype account:

    Go to the Skype account closure page: Skype Account Closure
    Sign in with the Microsoft account associated with your Skype.
    Follow the instructions to close your account.

For Telegram:

    Open the Telegram app on your device.
    Go to Settings (usually accessed by tapping on the three-line menu icon in the top left corner).
    In Settings, tap on your phone number.
    Tap on "Delete Number" or a similar option to remove your phone number from Telegram.

To deactivate your Telegram account:

    Go to Telegram's account self-destruction page: Delete My Account
    Enter your phone number in the international format.
    You'll receive a confirmation code via Telegram.
    Enter the confirmation code.
    Follow the prompts to deactivate your account.

Make sure to follow these steps carefully, as they'll ensure your phone number is removed and your accounts are deactivated.

**Comment by queenofcorgis** [comment]
Hi @caveman927 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Skype like [their community](https://answers.microsoft.com/en-us/skype/forum?sort=LastReplyDate&dir=Desc&tab=All&status=all&mod=&modAge=&advFil=&postedAfter=&postedBefore=&threadType=All&isFilterExpanded=true&page=1). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
